findimports.py

```diff
@@ -80,7 +80,7 @@
 from operator import attrgetter
 
 
-__version__ = '1.5.3.dev0'
+__version__ = '1.5.3.dev1'
```

Comment:
It is not necessary to increment the `.dev` suffix, but since it's a new feature, SemVer would suggest incrementing the minor version number:
```suggestion
__version__ = '1.6.0.dev0'
```